### PR TITLE
Refactor split button

### LINKED ISSUE #127: Create a split button component
Create a split button component for LanguageSource page
This issue is related to #124

### PR DESCRIPTION
**Pull request by xqiu1**
Fixes #127

### CODE REVIEW

File: src/app/components/AddSource.svelte [hunk 1]
```diff
@@ -3,6 +3,36 @@
...
+      text: "Upload",
+      isOutlined: false,
+      hasDropShadow: false,
+      dataCy: "add-sources-splitbtn-upload",
+      handleClick: uploadSourcesFromFile,
+    },
+    {
+      color: "grey",
+      size: "medium",
+      text: "Direct entry",
```
Comment (IanSeng) [inline src/app/components/AddSource.svelte:28]: Is this text supposed to be `Direct Entry `

**Review by IanSeng — APPROVED** [review]
LGTM

File: src/app/components/AddSource.svelte (hunk 1 above)
Comment (xqiu1) [reply src/app/components/AddSource.svelte:28]: The button text has already been set to uppercase letter, so it will still display as DIRECT ENTRY 😁

Comment (IanSeng) [reply src/app/components/AddSource.svelte:28]: awesome! I have totally forgotten it, thank you for clarifying it

**Pull request merged**